Algorithmia/CLI.py

```diff
@@ -245,6 +246,30 @@ def cat(self, path, client):
 
         return result
 
+    def compileAlgo(self, client, manifest_path="model_manifest.json"):
```

Comment:
I find the use of "compile" to mean "create lock file" confusing because we already use "compile" to mean "build" on the platform, and that isn't what this function is doing. Can we change to a word that actually indicates what the function does; something with the word "lock" in it? Perhaps `lockAlgo()` or `hashAlgo()`?